tools/__tasks__/compile/hash/index.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
         {
             description: 'Hash assets',
             task: () => {
-                const webpackRegex = /app-webpack/;
+                const webpackRegex = /[app-webpack|admin]/;
```

Comment:
feels like `admin` might be a bit too permissive for a regex? will this mean anything with that in the filename will be matched?